src/litserve/server.py

```diff
@@ -254,6 +254,7 @@ def inference_worker(
     stream,
 ):
     lit_api.setup(device)
+    lit_api.device = device
```

Comment:
should we have a single method to do both of these? maybe in future we can have `litapi.trigger_setup` which is mean to be used by LitServer to trigger methods.

Reply:
Sorry I didn't get this, can you clarify?

Reply:
my bad, I thought that I selected two lines while adding the comment. 
So right now, we call `litapi.setup` and then add set `litapi.device`, which kinda creates two source of truth. 

```
lit_api.setup(device)
lit_api.device = device
```

Reply:
saw your other comment, I think it won't matter when we remove device from setup.